DatadogInternal/Sources/Models/CrashReporting/DDCrashReport.swift

```diff
@@ -97,5 +100,6 @@ public struct DDCrashReport: Codable, PassthroughAnyCodable {
         self.meta = meta
         self.wasTruncated = wasTruncated
         self.context = context
+        self.additionalAttributes = AnyCodable(additionalAttributes)
```

Comment:
question/ Am I correct by saying that `Codable` requirement for `DDCrashReport` only comes from the message bus interface? We're using `PassthroughAnyCodable` so there's no actual coding and perf hit, but things like passing attribtues ☝️ are far from easy.

Reply:
Exactly, it is only for conformance.
The `DDCrashReport` and `CrashContext` are now both in `DatadogInternal`, so it's no longer required to pass them as `FeatureBaggage`. We could pass them directly as a `.crash(report: DDCrashReport, context: CrashContext)` message on the bus, but such change has wider impact and should be treated separately IMO.